src/app/calendar/ParcelCalendar.tsx

```diff
@@ -1,9 +1,18 @@
 "use client";
 
-import Calendar, { CalendarEvent } from "@/components/Calendar/Calendar";
+import Calendar from "@/components/Calendar/Calendar";
 import Title from "@/components/Title/Title";
 import React from "react";
-import styled from "styled-components";
+import styled, { useTheme } from "styled-components";
+import {
+    LocationColorMap,
+    parcelsToCollectionEvents,
+    parcelWithClientName,
+} from "@/app/calendar/parcelCalendarFunctions";
+
+interface ParcelCalendarProps {
+    parcelsWithCollectionDate: parcelWithClientName[];
+}
```

Comment:
Semicolon here?

Reply:
Prettier removes it so I think it is unnecessary